.github/workflows/ci.yml

```diff
@@ -5,10 +5,6 @@ on:
     branches: [ "main" ]
   pull_request: {}
 
-concurrency:
```

Comment:
This is a public repo where M$ pays for your github action runner minutes.  No need to economise. :)